packages/preview/glossarium/0.5.0/CHANGELOG.md

```diff
@@ -0,0 +1,67 @@
+# Changelog
+
+## Unreleased
```

Comment:
I don't know if this is intentional or not but there is no 0.5.0 section here.

Reply:
Ah yes, I forgot to update the changelog when I released and pushed the tag. I'll probably fix it for 5.0.1.